src/themes/default/text.ts

```diff
@@ -0,0 +1,19 @@
+import { TextTheme, ThemeHelpers } from "../types";
+
+export default (_theme: {}, { getColor }: ThemeHelpers): TextTheme => {
+  return {
+    getCSS: ({ color, align, wrap }) => {
+      return {
```

Comment:
remove return ? same as what you have in `src/themes/default/textStyles.ts`
https://github.com/moroshko/basis/pull/247/files#diff-5f026162d660728ff22e62a54e9e9e1046a66effbdb96cc9abe98dba2d9a6b9fR9

Reply:
It was like this before the change, see: https://github.com/moroshko/basis/pull/247/files/150f7c89fd2264570d1a9e2c6ada2cf2fc762cdd#diff-00a4e15ca1cf2270a4bfa851239d4d20ee055a94d0647c74a5abc84c6a431c39L4